javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/instrumentation/InstrumentationModuleInstaller.java

```diff
@@ -90,16 +90,19 @@ private AgentBuilder installIndyModule(
           setTypeMatcher(agentBuilder, instrumentationModule, typeInstrumentation)
               .transform(new PatchByteCodeVersionTransformer());
 
-      IndyTypeTransformerImpl typeTransformer =
-          new IndyTypeTransformerImpl(extendableAgentBuilder, instrumentationModule);
-      typeInstrumentation.transform(typeTransformer);
-      extendableAgentBuilder = typeTransformer.getAgentBuilder();
-      // TODO (Jonas): make instrumentation of bytecode older than 1.4 opt-in via a config option
       // TODO (Jonas): we are not calling
       // contextProvider.rewriteVirtualFieldsCalls(extendableAgentBuilder) anymore
       // As a result the advices should store `VirtualFields` as static variables instead of having
       // the lookup inline
       // We need to update our documentation on that
+      // Added back the call to contextProvider.rewriteVirtualFieldsCalls(extendableAgentBuilder)
+      // to get more instrumentations running.
+      extendableAgentBuilder = contextProvider.rewriteVirtualFieldsCalls(extendableAgentBuilder);
```

Comment:
Adding back `contextProvider.rewriteVirtualFieldsCalls` got some more tests passing (I don't remember which one, would need to retest). I don't know why it was removed.

Reply:
Makes sense. Part of what `rewriteVirtualFieldsCalls` was still necessary here, I split that out from it and called just that part.